### PR TITLE
[OSDOCS-5436]: Adding versioning info about HyperShift Operator

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.13

Issue: https://issues.redhat.com/browse/OSDOCS-5436

Link to docs preview: https://56603--docspreview.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#hosted-control-planes-version-support_control-plane

QE review:
- [x] QE has approved this change.

Additional information: This PR adds versioning information about the HyperShift Operator.

### CODE REVIEW

**Review by jparrill — COMMENTED** [review]
/lgtm

**Comment by jiezhao16** [comment]
lgtm

File: modules/hosted-control-planes-version-support.adoc [hunk 1]
```diff
@@ -12,7 +12,20 @@ With each major, minor, or patch version release of {product-title}, two compone
 * HyperShift Operator
 * Command-line interface (CLI)
 
-The HyperShift Operator manages the lifecycle of hosted clusters that are represented by `HostedCluster` API resources. The HyperShift Operator is released with each {product-title} release.
+The HyperShift Operator manages the lifecycle of hosted clusters that are represented by `HostedCluster` API resources. The HyperShift Operator is released with each {product-title} release. After the HyperShift Operator is installed, it creates a `ConfigMap` called `supported-versions` in the HyperShift namespace. That `ConfigMap` describes the HostedCluster versions that can be deployed, as shown in this example:
```
Comment (mburke5678) [inline modules/hosted-control-planes-version-support.adoc:15]: s/`ConfigMap`/`ConfigMap` object   --or-- config map

Should _as shown in this example_ be at the end of _it creates a `ConfigMap` called `supported-versions` in the HyperShift namespace_?

**Comment by mburke5678** [comment]
@lahinson 
A couple of nits. Otherwise LGTM

File: modules/hosted-control-planes-version-support.adoc (hunk 1 above)
Comment (lahinson) [reply modules/hosted-control-planes-version-support.adoc]: I updated the text according to your feedback. Thanks!

**Comment by ousleyp** [comment]
LGTM! Merging now.

**Pull request merged**